assets/scripts/lobby9.js

```diff
@@ -1926,4 +1931,19 @@ $(".maxNumPlayers").on("change", function(e){
     // console.log(e.target.value);
 
     socket.emit("update-room-max-players", e.target.value);
-});
\ No newline at end of file
+});
+
+function getGunPos(){
+    if (docCookies.hasItem('gunPos.x') === true && docCookies.hasItem('gunPos.y') === true) {
```

Comment:
Shouldn't need to access cookies, just check whether the option is ticked or not. `$("#<checkbox>").checked === true` <- from the top of my mind.

Reply:
Accepted. Will be resolved in the next commit